<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agregar_MainWindow</class>
 <widget class="QMainWindow" name="Agregar_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>479</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>762</width>
    <height>479</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>762</width>
    <height>479</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar Pélicula</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Imagenes/logos-UNAH-600x600.png</normaloff>Imagenes/logos-UNAH-600x600.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(204, 204, 204);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Nombre_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>681</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:black;
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
}
QLineEdit::{
sizeHint(10);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>Nombre de Pélicula</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Duracion_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>681</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:black;
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
}
QLineEdit::{
sizeHint(10);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="placeholderText">
     <string>Duración de la Pélicula(HH:MM:SS)</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Director_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>681</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color:black;
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
}
QLineEdit::{
sizeHint(10);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>Director de la Pélicula</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>681</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentText">
     <string>----Seleccione un Género----</string>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <item>
     <property name="text">
      <string>----Seleccione un Género----</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hechos Reales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bélicas </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Comedias Musicales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Acción</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Artes marciales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aventuras</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ciencia ficción </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Comedia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dibujos animados </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Documental </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Espada y Hechicería </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Espionaje</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Horror</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Misterio </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Muertos vivientes </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Propaganda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suspenso</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Terror</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Deportivas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dramáticas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fantásticas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Infantiles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Musicales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Policíacas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Psicológicas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Románticas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobre animales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobre aviación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobre delincuencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobre discapacitados</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobre religión</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobre política</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Agregar_pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 174, 255);
font: 10pt &quot;Hack&quot;;
border-style: solid;
border-color: black;
border-width: 0.5px;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancelar_pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 115, 115);
font: 10pt &quot;Hack&quot;;
border-style: solid;
border-color: black;
border-width: 0.5px;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="shortcut">
     <string notr="true"/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="Descripcion_textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>681</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: 10pt &quot;Hack&quot;;
border-width: 3px;
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Descripción de la Pélicula</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
